bashflux/cmd/channels.go

```diff
@@ -0,0 +1,179 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/mainflux/mainflux/clients"
+	"github.com/spf13/cobra"
+)
+
+var chanEndPoint = "channels"
+
+// Channels
+var CmdChannels = &cobra.Command{
```

Comment:
Based on the remark given above, you can do the following (draft code):

```golang
package clients

var commands :=[]cobra.Command{
    cobra.Command{ /* specify create */},
    cobra.Command{/* specify retrieve */},
    // etc.
}

// New does what godoc says...
func New() *cobra.Command {
    // package root
    cmd := cobra.Command { ... }

    for _, c := range commands {
        cmd.AddCommand(c)
    }

    return &cmd
}
```

The same "pattern" should be applied for each API component.